src/Repository/FileRepository.hpp

```diff
@@ -24,6 +24,7 @@ Copyright_License {
 #ifndef XCSOAR_FILE_REPOSITORY_HPP
 #define XCSOAR_FILE_REPOSITORY_HPP
 
+#include "OS/Path.hpp"
```

Comment:
This adds a heavy header dependency, but this should better be a forward declaration.